_topic_maps/_topic_map_rosa.yml

```diff
@@ -606,4 +606,4 @@ Topics:
 - Name: Troubleshooting cluster deployments
   File: rosa-troubleshooting-deployments
 - Name: Red Hat OpenShift Service on AWS managed resources
-  File: rosa-managed-resources
```

Comment:
You need the end of file marker. Will you check your editor settings to make sure it's not stripping it?

Reply:
Marked the end of file.